ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewGroupDrawingOrderHelper.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.react.uimanager;
+
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.facebook.react.views.view.ReactViewManager;
```

Comment:
use `ViewGroupManager.getViewZIndex` instead of `ReactViewManager.getViewZIndex` to fix dep issue